TestViewer/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,33 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
```

Comment:
Are you using the automatic version generation system that toolkit and Lottie use? If not you probably should. If you are then something is not set up right because it should be generating some of this info for you.

Reply:
What was the resolution?

---
In reply to: [280247999](https://github.com/windows-toolkit/SceneLoader/pull/1#discussion_r280247999) [](ancestors = 280247999)

Reply:
Yes I am but it's not working properly. I will fix it in a future update.

---
In reply to: [280957845](https://github.com/windows-toolkit/SceneLoader/pull/1#discussion_r280957845) [](ancestors = 280957845,280247999)